docs/features/datastores/overview.md

```diff
@@ -0,0 +1,31 @@
+# Demystifying Datastorses
+
+## What are Datastores
+
+Datastores are high-performance, low-latency, versioned, and scalable datasets.
+
+## What files can be accepted
+Datastores can be created from any file type
```

Comment:
So there are filetypes and then there are upload methods. We should point out both - so for example HTTP URL to compressed directories tar.gz and zip folders

Reply:
All I know for upload methods are http and s3 buckets. Feel free to update with your own commit addressing this request as there may be more.